internal/scm/gitlab/gitlab.go

```diff
@@ -226,6 +227,7 @@ func (g *Gitlab) CreatePullRequest(ctx context.Context, repo scm.Repository, prR
 		TargetProjectID:    &r.pid,
 		ReviewerIDs:        reviewersIDs,
 		RemoveSourceBranch: &removeSourceBranch,
+		Squash:				&squash,
```

Comment:
Weird, the online VS code does not show this formatting 🤔 

![image](https://user-images.githubusercontent.com/17970041/152336471-ecb30071-d1b3-4d31-9927-9833c5898f13.png)